Include/dynamic_annotations.h

```diff
@@ -437,6 +455,7 @@ void AnnotateEnableRaceDetection(const char *file, int line, int enable);
 void AnnotateNoOp(const char *file, int line,
                   const volatile void *arg);
 void AnnotateFlushState(const char *file, int line);
+#endif  /*ABSL_HAVE_THREAD_SANITIZER*/
```

Comment:
change line 422
```
void AnnotateBenignRaceSized(const char *file, int line,
                        const volatile void *address,
                        long size,
                        const char *description);
```

to 

```suggestion
/// proton: starts.
void AnnotateBenignRaceSized(const char *file, int line,
                        const volatile void *address,
                        size_t size,
                        const char *description);
/// proton: ends.
```